src/components/Controls/Int.vue

```diff
@@ -4,11 +4,16 @@ import Data from './Data';
 export default {
   name: 'Int',
   extends: Data,
+  computed: {
+    inputType() {
+      return 'number';
+    },
```

Comment:
The new number inputs have added the up/down arrows for the fields, They seem less useful to me and we can hide them through css.